corehq/doctypemigrations/migrator_instances.py

```diff
@@ -36,6 +36,23 @@
     doc_types=('Domain',)
 )
 
+apps_migration = Migrator(
+    slug='apps',
+    source_db_name=None,
+    target_db_name=settings.NEW_DOMAINS_DB,
+    doc_types=(
+        'Application',
+        'ApplicationBase',
+        'CareplanConfig',
+        'DeleteApplicationRecord',
+        'DeleteFormRecord',
+        'DeleteModuleRecord',
+        'RemoteApp',
+        'SavedAppBuild',
+        'VersionedDoc',
```

Comment:
Do I need to include "abstract" doc types like "ApplicationBase"?  It's not intended to be written directly, but you can totally do `ApplicationBase().save()`